.circleci/config.yml

```diff
@@ -385,7 +311,7 @@ commands:
             scripts/configure_dev.sh
             scripts/buildtools/install_buildtools.sh -o "gotest.tools/gotestsum"
             export ALGOTEST=1
-            export SHORTTEST=-short
+            export SHORTTEST=<< parameters.go_test_flags >>
```

Comment:
since it can now be any kind of flags, shouldn't the left side change also (from SHORTTEST to TESTFLAGS or something?)

Reply:
SHORTTEST is an environment variable used in 1run_integration_tests.sh1 which I think calls `make` I think I want to avoid modifying existing code right now.

Reply:
Hmmm.. but you're right that doesn't make as much sense in this context as when it's called in the gotestsum command, so I think I should rename the parameters.go_test_flags to just parameters.short_test_flag or something. Thanks!